README.md

```diff
@@ -1,4 +1,4 @@
-# tags-thrift-schema
+# tags-thrift-schema [![Release](https://github.com/guardian/tags-thrift-schema/actions/workflows/release.yml/badge.svg)](https://github.com/guardian/tags-thrift-schema/actions/workflows/release.yml)
```

Comment:
Also cool to add in a badge showing the level of Scala support, as added to the Scaladex in https://github.com/scalacenter/scaladex/pull/660 !

![image](https://github.com/guardian/tags-thrift-schema/assets/52038/6d7a318d-7f17-42a6-a044-038b56603339)


```suggestion
# tags-thrift-schema
[![tags-thrift-schema Scala version support](https://index.scala-lang.org/guardian/tags-thrift-schema/tags-thrift-schema/latest-by-scala-version.svg?platform=jvm)](https://index.scala-lang.org/guardian/tags-thrift-schema/tags-thrift-schema)
[![Release](https://github.com/guardian/tags-thrift-schema/actions/workflows/release.yml/badge.svg)](https://github.com/guardian/tags-thrift-schema/actions/workflows/release.yml)
```